<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AjouterProduit</class>
 <widget class="QWidget" name="AjouterProduit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>514</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">

QWidget {
background: white;
}
QLabel {
color: black;
font-weight: bold;
}



#groupBox {
border: 3px solid gray;
border-radius: 40px;
background: white;
background: QLinearGradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #eef, stop: 1 #ccf);
}

QPushButton {
color: white;
background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #88d, stop: 0.1 #99e, stop: 0.49 #77c, stop: 0.5 #66b, stop: 1 #77c);
border-width: 1px;
border-color: #339;
border-style: solid;
border-radius: 7;
padding: 3px;
padding-left: 5px;
font-weight: bold;
font-size: 13px;
padding-right: 5px;
min-width: 90px;
max-width: 90px;
min-height: 12px;
max-height: 16px;

border: 2px solid #555;
border-radius: 11px;
padding: 5px;
max-width: 150px;
min-width: 100px;
}

QLineEdit {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}

QComboBox {
padding: 1px;
border-style: solid;
border: 2px solid gray;
border-radius: 8px;
}
</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QPushButton" name="ajouter">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>380</y>
      <width>121</width>
      <height>26</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajouter</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Paramètres Théoriques</string>
    </property>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>173</width>
       <height>110</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>EST</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEST"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>GS</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineGS"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>PH</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="linePH"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>T° Cible</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineT"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>19</y>
      <width>251</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QComboBox" name="comboPr">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>10</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>12</y>
       <width>52</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Produit</string>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="linerecette">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>61</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>62</y>
      <width>55</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Recette</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>120</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Dernière Recette :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="laberecette">
       <property name="text">
        <string>-- / --</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableView" name="tabproduit">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>441</width>
     <height>431</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="retour">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>450</y>
     <width>114</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QPushButton" name="valideraj">
   <property name="geometry">
    <rect>
     <x>607</x>
     <y>450</y>
     <width>114</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
